.github/workflows/deploy-sandbox.yml

```
@@ -0,0 +1,45 @@
+name: deploy to prod
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)